Bentley/k9 threat intel (#21246)
DataDog/documentation: content/en/security/threat_intel.md

| malware | malicious | application library versions, file hashes | Malicious packages and communication with mining pools| ASM, CWS |
| scanner | suspicious | IP addresses | Reputation for scanners | ASM, Cloud SIEM |
| Tor | suspicious | IP addresses  | Corporate policy violations for user activity | ASM and Cloud SIEM |

### Threat Intelligence Intents
| Intent | Use Case |
|--------|----------|
| benign | Corporate VPNs and informational enrichments |
| suspicious | Low reputation | 
| malicious | Malicious reputation | 

## Entity Types
| Entity Type | Example | Use Cases | 
|-------------|---------|-----------------------------|
| IP addresses | 128.66.0.1 | Identify IP addresses associated with attacks, command and control, and scanning activity | 
| domains | example.com, subdomain.example.com | Domains associated with malicious use. Often used with malware as a command and control |
| application packages versions | (example_package, 1.0.0) | Identify malicious packages downloaded from PyPi |
| file hashes [SHA1, SHA256] | 5f7afeeee13aaee6874a59a510b75767156f75d14db0cd4e1725ee619730ccc8 | Identify a distinct file associated with malware or compromise |

> Threat intelligence sources and categories are not customer configurable at this time. For specific product implementations see:
> 
> [ASM](https://docs.datadoghq.com/security/application_security/threats/threat-intelligence/)
> 

Reviewer: ```suggestion
Threat intelligence sources and categories are not customer configurable at this time. 

# Product implementations

## ASM

- [Threat Intelligence in ASM](https://docs.datadoghq.com/security/application_security/threats/threat-intelligence/)
```

Minor point, but if we have this info as a heading, it will be visible in the document outline. If it's visible in the outline, people will immediately know that there are product-specific nuances worth checking out.

Whereas now it's a bit hidden.

With threat intelligence, reputation is key, but it must be weighed alongside other evidence. Relying solely on IP and Domain intelligence for blocking traffic is not recommended, with few exceptions. A balanced, evidence-based approach is essential.

Threat intelligence used in Detection Rules should reference the Datadog keys such as category and intent. Other keys should not be used.

## Transparency in Threat Intelligence 
Datadog ensures transparency by providing external links to external threat intelligence sources associated with a detection. All external threat intelligence at Datadog is ingested to the Datadog platform for enrichment and detection. Datadog does not send customer data to threat intelligence sources.

The detections and enrichments are accessible in the UI and event JSON.

## Threat Intel Facets
Sources, categories, and intents are available as facets and filters on relevant product Explorers. 

### Threat Intel Sources

| Source | Categories | Source Use Cases | Primary Products | 
|--------|------------|-----------|------------------|
| Datadog Threat Research| scanners, exploits | Honeypots focused on software specific threats | ASM and CWS |

> | [Datadog Guarddog](https://github.com/DataDog/guarddog) | malware | Malware in application libraries | ASM |

Reviewer 2: Guarddog is not yet available, will be added in Q1 (part of our OKRs) we can remove this line now and add it later when its actually available.

Author: updated thank you.

| [Spur](https://spur.us/) | residential_proxy | Proxies associated with fraud and credential stuffing | ASM and Cloud SIEM |

> | [Spur](https://spur.us/) | malware_proxy | Proxies associated with malware command and control | ASM and CWS |

Reviewer 2: Spur is not yet licensed to be used in CWS, no?

Author: updated. thank you.
